src/deployment/deploylib/registration.py

```diff
@@ -745,12 +746,20 @@ def main() -> None:
 
     subparsers = parser.add_subparsers(title="commands", dest="command")
     subparsers.add_parser("update_pool_registration", parents=[parent_parser])
-    role_assignment_parser = subparsers.add_parser(
+    scaleset_role_assignment_parser = subparsers.add_parser(
         "assign_scaleset_role",
         parents=[parent_parser],
     )
-    role_assignment_parser.add_argument(
-        "scaleset_name",
+    scaleset_role_assignment_parser.add_argument(
+        "--scaleset_name",
+        help="the name of the scaleset",
+    )
+    cli_role_assignment_parser = subparsers.add_parser(
+        "assign_cli_role",
```

Comment:
should this one be --assign_cli_role as well then ?

Reply:
Sorry Stas, what are you referring to?